packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -328,8 +328,8 @@ const messages = {
     feide: 'Denne ressursen er bare tilgjengelig for lærere som er pålogget med Feide.',
     resources:
       'Dette er ikke et komplett læremiddel produsert av NDLA, men ei ressurssamling som vi håper kan være nyttig for deg.',
-    beta: 'Dette faget er i betaversjon.',
-    outdatedSubject: 'Dette emnet hører til et fag som ikke er oppdatert etter gjeldende læreplan.',
+    subjectOutdated: 'Dette faget følger en utgått læreplan.',
+    subjectBeta: 'Dette faget er i beta og vil være klart til skolestart.',
```

Comment:
Burde det stå betaversjon også på bokmål?

Reply:
Disse er dratt rett ut fra språkdokumentet de linket til i Trello. Kan godt endre det til betaversjon hvis du vil!

Reply:
Tenkte konsistens var viktig, men då må vi spørre på trello igjen.

Reply:
Sjekka dokumentet igjen og det står betaversjon der. Trur også at den korte varianten er best i tilfelle enkelte fag likevel ikkje er klare til skulestart.
```suggestion
    subjectBeta: 'Dette faget er i betaversjon.',
```